src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+server.error.whitelabel.enabled=false
```

Comment:
What does this do?

Reply:
it hides this
![](https://www.logicbig.com/tutorials/spring-framework/spring-boot/disable-default-error-page/images/output1.png)

but that's better than tomcat
![](https://www.logicbig.com/tutorials/spring-framework/spring-boot/disable-default-error-page/images/tom-exclude.png)

so I'm going to revert this change.